packages/pageflow/src/editor/base.js

```diff
@@ -17,11 +17,12 @@ export const startEditor = function(options) {
 
   $(function() {
     $.when(
-      $.getJSON('/editor/entries/' + options.entryId + '/seed'),
-      pageflow.browser.detectFeatures()
+      $.getJSON('/editor/entries/' + options.entryId + '/seed')
     )
       .done(function(result) {
-        app.start(result[0]);
+        pageflow.browser.detectFeatures().then(function () {
+          app.start(result);
+        });
       })
```

Comment:
you mentioned in previous comments that app needs to start after browser detect features

Reply:
I think you are referring to my comment here: https://github.com/codevise/pageflow/pull/1370#discussion_r404194359. But that was only since the previous change moved feature detection after start. The code as it was already did feature detection before start (only in parallel to loading the seed - now it's done sequentially). I think no change is needed here at all.

Reply:
you mean this needs to be reverted back to as it was? or we can keep it like this?

Reply:
I mean this should be reverted to the way it is on `master` since I do not see a need to change anything here.